lib/log.h

```diff
@@ -92,9 +92,9 @@ enum {
  *                 content, e.g. SKB queues, rbtree, etc.
  */
 #define __BNR		"[tempesta " BANNER "] "
-#define __T_DBG1(...) 	pr_debug(__BNR "  " __VA_ARGS__)
-#define __T_DBG2(...) 	pr_debug(__BNR "    " __VA_ARGS__)
-#define __T_DBG3(...)	pr_debug(__BNR "      " __VA_ARGS__)
+#define __T_DBG1(...) 	printk(__BNR "  " __VA_ARGS__)
```

Comment:
Why do change `pr_debug` to `printk`. I would prefer to use `pr_debug` or pass message level to `printk`. For dynamic debug we can define another macro that will not excluded for build even if `DEBUG` is not set.